### ISSUE TITLE
One-shot Conversion

### ISSUE THREAD

**Issue by sbkim052** [mindslab-ai/cotatron#6]
Is this work on the one-shot voice conversion environment?

**Comment by seungwonpark** [comment]
TL; DR: No.

To make things sure, let's make sure that we have the same definition:
- Target speaker: The converted speech should have a target speaker's speaker identity. In other words, the converted speech should sound as if the speech was spoken by a target speaker.
- One-shot voice conversion: The voice conversion is possible even when the target speaker is not seen during training. (Such as [One-shot Voice Conversion by Separating Speaker and Content Representations with Instance Normalization](https://arxiv.org/abs/1904.05742))

With Cotatron, it is possible to use source speech from an unseen speaker, however, the target speaker should have been seen during training. In other words, any-to-many is possible, but not any-to-any.

Though, if we utilize a pre-trained speaker verification network to represent the speaker's identity, we think any-to-any conversion should be possible, so the one-shot voice conversion.

**Comment by sbkim052** [comment]
Thank you so much.

**Comment by sbkim052** [comment]
@seungwonpark 
Quick question about the demo.

For the bonus converted samples(korean voice conversion),  have you trained the model with the korean datasets or trained it only with VCTK and tested with korean speech?

**Comment by seungwonpark** [comment]
For the bonus Korean sample that you've mentioned, the system trained with a proprietary Korean speech dataset was used.

**Comment by sbkim052** [comment]
Thanks:)

**Comment by ghost** [comment]
How many speaker in korean DB? How long speech each speaker? how many epoch encoder? how many epoch vocoder? Thanks you
